src/plugins/messageservices/imap/imapclient.cpp

```diff
@@ -536,12 +423,14 @@ ImapClient::ImapClient(QObject* parent)
             this, SLOT(downloadSize(QString, int)) );
     connect(&_protocol, SIGNAL(urlAuthorized(QString)),
             this, SLOT(urlAuthorized(QString)) );
-    connect(&_protocol, SIGNAL(folderCreated(QString)),
-            this, SLOT(folderCreated(QString)));
-    connect(&_protocol, SIGNAL(folderDeleted(QMailFolder)),
-            this, SLOT(folderDeleted(QMailFolder)));
-    connect(&_protocol, SIGNAL(folderRenamed(QMailFolder, QString)),
-            this, SLOT(folderRenamed(QMailFolder, QString)));
+    connect(&_protocol, SIGNAL(folderCreated(QString, bool)),
```

Comment:
This is to handle 'failure' cases as well (e.g. decrease internal outstanding requests counters)